fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/AbstractPlan.java

```diff
@@ -77,6 +77,10 @@ public Optional<GroupExpression> getGroupExpression() {
         return groupExpression;
     }
 
+    public void setGroupExpression(GroupExpression groupExpression) {
+        this.groupExpression = Optional.of(groupExpression);
+    }
+
```

Comment:
plan should be immutable so can not support setXxx. You can invoke `withGroupExpression()`